flank-scripts/src/main/kotlin/flank/scripts/release/hub/RemoveOldTag.kt

```diff
@@ -0,0 +1,16 @@
+package flank.scripts.release.hub
+
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.extensions.authentication
+import flank.scripts.exceptions.mapClientError
+import flank.scripts.exceptions.toGithubException
+
+fun deleteOldTag(tag: String, username: String, password: String) =
```

Comment:
Rename file to `DeleteOldTag.kt` to match to function name

Reply:
changed